SevenBridges/TotalSegmentator/splitWorkflow/threeVM.cwl

```diff
@@ -211,13 +211,17 @@ steps:
       sbg:modifiedBy: vamsikrishna14
       sbg:modifiedOn: 1681840727
       sbg:revisionNotes: ''
+    - sbg:revision: 3
+      sbg:modifiedBy: vamsikrishna14
+      sbg:modifiedOn: 1695396114
+      sbg:revisionNotes: ''
     sbg:image_url:
     sbg:appVersion:
     - v1.2
-    sbg:id: vamsikrishna14/idc/downloaddicomandconvert/2
-    sbg:revision: 2
+    sbg:id: vamsikrishna14/idc/downloaddicomandconvert/3
```

Comment:
Since an issue was opened just to address references, I created another branch for it, fixed most references already there, and plan to submit a pull request.

for the personal ID, I did not evaluate what part of the cwl file is mandatory and what is not. I will evaluate, clean up and submit the pull request fix referencing #4.